node/node.go

```diff
@@ -300,6 +302,9 @@ func (node *AlgorandFullNode) Start() {
 	node.mu.Lock()
 	defer node.mu.Unlock()
 
+	// initialize last round start time to current time
+	node.lastRoundTimestamp = time.Now()
```

Comment:
Please consider removing this line. It implies incorrect data.

Reply:
Removed.